README.md

```diff
@@ -105,13 +105,15 @@ Follow instructions [here](https://github.com/microsoft/vscode-apimanagement/iss
 
 ## Managing Azure Subscriptions
 
-If you are not signed in to Azure, you will see a "Sign in to Azure..." link. Alternatively, you can select "View->Command Palette" in the VS Code menu, and search for "Azure: Sign In".
+If you are not signed in to Azure, you will see a "Sign in to Azure..." link.
 
-If you don't have an Azure Account, you can sign up for one today for free and receive $200 in credits by selecting "Create a Free Azure Account..." or selecting "View->Command Palette" and searching for "Azure: Create an Account".
+If you don't have an Azure Account, you can sign up for one today for free and receive $200 in credits by selecting "Create a Free Azure Account...", or select "Create an Azure for Students Account...".
 
-You may sign out of Azure by selecting "View->Command Palette" and searching for "Azure: Sign Out".
+You may sign out of Azure from VSCode account, select your Azure Account in account page and click "Sign Out".

```

Comment:
To sign out of Azure in VSCode, go to the account page, select your Azure account, and click 'Sign Out'.

Reply:
updated, thanks